crates/solver/src/liquidity/zeroex.rs

```diff
@@ -194,21 +194,21 @@ fn get_useful_orders(order_buckets: OrderBuckets, orders_per_type: usize) -> Vec
         // best priced orders are those that have the maximum maker_amount /
         // taker_amount ratio
         orders.sort_by(|order_1, order_2| {
-            let price_1 = order_1.order.maker_amount as f64 / order_1.order.taker_amount as f64;
-            let price_2 = order_2.order.maker_amount as f64 / order_2.order.taker_amount as f64;
+            let price_1 = order_1.order().maker_amount as f64 / order_1.order().taker_amount as f64;
+            let price_2 = order_2.order().maker_amount as f64 / order_2.order().taker_amount as f64;
             price_1.total_cmp(&price_2)
         });
         filtered_zeroex_orders.extend(orders.drain(orders.len() - orders_per_type..));
 
-        orders.sort_by_key(|order| order.metadata.remaining_fillable_taker_amount);
-        filtered_zeroex_orders.extend(orders.into_iter().rev().take(orders_per_type));
+        orders.sort_by_key(|order| order.metadata().remaining_fillable_taker_amount);
+        filtered_zeroex_orders.extend(orders.iter().rev().take(orders_per_type).cloned());
```

Comment:
Why to do this change in this line? eve tho the clone is cheap, but there is no need for it :thinking:

Reply:
Looks like a leftover from one of the previous states of the PR. Reverted, thanks.